app/models/katello/concerns/widget_extensions.rb

```diff
@@ -0,0 +1,23 @@
+module Katello
+  module Concerns
+    module WidgetExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        SUBSCRIPTION_TEMPLATES = %w[subscription_status_widget subscription_widget].freeze
+
+        scope :remove_subscription_widgets, -> { where.not(template: ::Widget::SUBSCRIPTION_TEMPLATES) }
```

Comment:
```suggestion
        scope :without_subscription_widgets, -> { where.not(template: ::Widget::SUBSCRIPTION_TEMPLATES) }
```
`remove` to me implies an action, but in reality this is more of a filter.